### PR TITLE
Install and Uninstall doc

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

## Description

This PR establishes documentation on how to install and uninstall InSpec via a variety of operating systems and methods. The following sections include:
- [x] Install
    - [x] macOS
    - [x] Windows
    - [x] Linux
- [x] Uninstall
    - [x] macOS
    - [x] Windows
    - [x] Linux

## Related Issue
Addresses https://github.com/inspec/inspec/issues/1338

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+You can find [Chef InSpec on the Depot here](https://bldr.habitat.sh/#/pkgs/chef/inspec).
+
+### Linux
+
+#### CLI
+
+For Ubuntu, use the following command to install Chef InSpec:
+
+```
+sudo dpkg -i /path/to/inspec.deb
```
Comment (mjingle) [inline docs/install.md:67]: Note there is an alternate command mentioned in the InSpec Readme:
> RedHat, Ubuntu, and macOS
> curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec

File: docs/install.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+```
+
+For Red Hat Enterprise Linux, use the following command to install Chef InSpec:
+
+```
+sudo rpm -U /path-to/inspec.rpm
```
Comment (mjingle) [inline docs/install.md:73]: See alternate command note above, which is applicable.

File: docs/install.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+```
+
+#### Habitat
+
+Use Chef Habitat to install Chef InSpec, or include Chef InSpec in your own Chef Habitat packages. Chef InSpec is available [on the Habitat Builder Depot](https://bldr.habitat.sh/#/pkgs/chef/inspec).
+
+Use the following command to install the Chef InSpec package:
+
+```
+$ hab pkg uninstall chef/inspec
```
Comment (zenspider) [inline docs/install.md]: "uninstall" -> "install"

File: docs/install.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+#### CLI
+
+You can download Chef InSpec via script:
+```
+curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec
+```
+
+#### Habitat
+
+Use Chef Habitat to install Chef InSpec, or include Chef InSpec in your own Chef Habitat packages. Chef InSpec is available [on the Habitat Builder Depot](https://bldr.habitat.sh/#/pkgs/chef/inspec).
```
Comment (zenspider) [inline docs/install.md]: Looks to me like this is NOT a valid option for mac.

File: docs/install.md [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+You can find [Chef InSpec on the Depot here](https://bldr.habitat.sh/#/pkgs/chef/inspec).
+
+### Linux
+
+#### CLI
+
+For Ubuntu, use the following command to install Chef InSpec:
+
+```
+sudo dpkg -i /path/to/inspec.deb
```
Comment (zenspider) [inline docs/install.md:67]: Where are these tho?

File: docs/install.md [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+
+Use the following *destructive* command to remove the Chef InSpec standalone Homebrew package:
+
+```
+$ brew uninstall --force chef/chef/inspec
+```
+
+#### CLI
+
+Ue the following *destructive* command in your terminal to remove the Chef InSpec package:
```
Comment (zenspider) [inline docs/install.md]: "Use"

File: docs/install.md [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+Chef InSpec can be uninstalled using the steps below that are appropriate for the method of Chef InSpec installation.
+
+### macOS
+
+#### Homebrew
+
+Use the following *destructive* command to remove the Chef InSpec standalone Homebrew package:
+
+```
+$ brew uninstall --force chef/chef/inspec
```
Comment (zenspider) [inline docs/install.md]: Chances are, if you needed to `sudo` under the covers to install, you'll prolly need to do so to uninstall?

File: docs/install.md [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+```
+$ sudo rm -rf /opt/inspec
+```
+
+#### Habitat
+
+To uninstall the Chef InSpec package, use the following command:
+
+```
+$ hab pkg uninstall chef/inspec
```
Comment (zenspider) [inline docs/install.md]: again, I don't think this is valid for osx (yet?)

File: docs/install.md [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
+```
+
+See about available flag options in the [hab pkg uninstall documentation](https://www.habitat.sh/docs/habitat-cli/#hab-pkg-uninstall).
+
+
+### Windows
+
+#### Installer
+
+Use *Add / Remove Programs* to remove Chef InSpec.
```
Comment (zenspider) [inline docs/install.md:109]: As a non-windows user... I wouldn't easily know how to do this.

**Review by zenspider — CHANGES_REQUESTED** [review]

File: docs/install.md (hunk 4 above)
Comment (miah) [reply docs/install.md]: Habitat for Mac runs Linux under the covers, so it technically is a valid option for Mac even though the platform isn't listed on bldr.

File: docs/install.md (hunk 9 above)
Comment (miah) [reply docs/install.md:109]: I can help write up a list of steps to get to Add / Remove Programs.

File: docs/install.md (hunk 5 above)
Comment (mjingle) [reply docs/install.md:67]: What are the "these" you speak of?

File: docs/install.md (hunk 7 above)
Comment (mjingle) [reply docs/install.md]: I'm somewhat confused by this comment. The content above is adapted from the homebrew documentation.

File: docs/install.md (hunk 8 above)
Comment (mjingle) [reply docs/install.md]: Not an issue anymore :)

File: docs/install.md (hunk 4 above)
Comment (mjingle) [reply docs/install.md]: Not an issue anymore :)

File: docs/install.md (hunk 3 above)
Comment (mjingle) [reply docs/install.md]: Good catch! Not an issue anymore :)

**Review by clintoncwolfe — APPROVED** [review]
Thanks!

**Comment by mjingle** [comment]
@inspec/inspec-core-team Content is complete for this PR. If there are changes needed for this content, please open a new PR addressing this.

This PR also includes docs navigation changes. 1) The install/uninstall page (install.md) is established. 2) I also changed the /downloads button to redirect to the install/uninstall page. Please check first to see if this breaks the website or not.

Content for the downloads page is still present in the repo, just not directly exposed. Will address this in the new year.

File: docs/install.md (hunk 9 above)
Comment (mjingle) [reply docs/install.md:109]: @miah Windows documentation on Add/Remove Programs does exist, but it doesn't have a permalink that I could find. Let's work on getting this additional content in via the new year and a new PR. TIA!

File: docs/install.md (hunk 7 above)
Comment (mjingle) [reply docs/install.md]: Also, I had to triple check and this command did not work. I updated the PR accordingly to something that does work! :)

**Review by kekaichinose — APPROVED** [review]
Looks good to me!

File: docs/install.md (hunk 1 above)
Comment (zenspider) [reply docs/install.md:67]: Given that the alternative is downloading ~800 loc to run on the fly (such that any sane/wary person would want to read and audit that code), I think it is better to push people towards a managed package where possible. `curl ... | bash ...`  should be a last resort.

File: docs/install.md (hunk 2 above)
Comment (zenspider) [reply docs/install.md:73]: I think a simple table might be easier to digest:

| OS | Command |
| ------- | ------------- |
| Ubuntu | `sudo dpkg -i /path/to/inspec.deb` |
| RHEL | `sudo rpm -U /path-to/inspec.rpm` |
| SUSE    | `zypper install /path-to/inspec.rpm` |

(and probably sudo on zypper?)

File: docs/install.md [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+
+```
+sudo rpm -e inspec
+```
+
+For SUSE Linux Enterprise Server, use the following *destructive* command to uninstall Chef InSpec: 
+
+```
+zypper remove inspec
+```
```
Comment (zenspider) [inline docs/install.md:133]: Same table suggestion here... Maybe across all OSes.

File: docs/install.md [hunk 11]
```diff
@@ -0,0 +1,133 @@
...
+title: Install and Uninstall
+---
+# Install and Uninstall
+
+Users can choose between operating systems of MacOS, Windows, and Linux for Chef InSpec.
+
+## Install Chef InSpec
+
+You can download the latest Chef InSpec package relevant to your operating system at [our Downloads Page](https://downloads.chef.io/inspec).
+
```
Comment (zenspider) [inline docs/install.md:11]: Might be nice to bring back the button for here as well as the clipboard copy links for the commands below.

See https://www.inspec.io/downloads/

File: docs/install.md (hunk 2 above)
Comment (miah) [reply docs/install.md:73]: Yes, you'll need sudo with zypper too.

**Review by miah — CHANGES_REQUESTED** [review]
Looks great @mjingle. Just a few minor changes requested in here.

File: docs/install.md (hunk 2 above)
Comment (mjingle) [reply docs/install.md:73]: For accessibility and site migration reasons, a table will not be included. Sudo will be included with zypper! Thanks!

File: docs/install.md (hunk 10 above)
Comment (mjingle) [reply docs/install.md:133]: zypper will be updated with sudo here.

File: docs/install.md (hunk 11 above)
Comment (mjingle) [reply docs/install.md:11]: Understood. I'm not going to be able to hardcode a button in this first iteration, and it probably won't play well with a site migration. I do like those features though!

**Review by zenspider — APPROVED** [review]

**Review by miah — APPROVED** [review]

**Pull request merged**